docs/index.md

```diff
@@ -19,7 +18,7 @@ The analyzer currently provides the following diagnostics:
 |----------------|------------------------------------------|-----------|-------------------------------------------------------------------------------------------------------------------------------------------------------|
 | FakeItEasy0001 | Unused call specification                | no        | Triggered when you specify a call but don't configure or assert it, making it a no-op.                                                                |
 | FakeItEasy0002 | Non-virtual member configuration         | no        | Triggered when you try to configure a non-virtual member, which cannot be faked.                                                                      |
-| FakeItEasy0003 | Argument constraint outside call spec    | no        | Triggered when you try to use an [argument constraint](argument-constraints.md) outside of a [call specification](specifying-a-call-to-configure.md). |
-| FakeItEasy0004 | Argument constraint nullability mismatch | yes       | Triggered when you use a non-nullable [argument constraint](argument-constraints.md) for a nullable parameter. Calls where the argument is null won't be matched. If this is intentional, consider using `A<T?>.That.IsNotNull()` instead. If it's not, make the argument constraint nullable (`A<T?>`). |
-| FakeItEasy0005 | Argument constraint type mismatch        | yes       | Triggered when you use an [argument constraint](argument-constraints.md) whose type doesn't match the parameter. No such calls can be matched.        |
+| FakeItEasy0003 | Argument constraint outside call spec    | no        | Triggered when you try to use an [argument constraint](https://fakeiteasy.readthedocs.io/en/latest/argument-constraints/) outside of a [call specification](https://fakeiteasy.readthedocs.io/en/latest/specifying-a-call-to-configure/). |

```

Comment:
I'm not yet sure if there's a better way to refer to links in the parent project. Given version drift between the two, maybe this is the best we can do. I think it's worth trying for now.

Reply:
I'm a little disappointed there's not a mechanism for inter-project links. Well, I guess we can live with it.